services/apps/alcs/src/alcs/compliance-and-enforcement/compliance-and-enforcement.service.ts

```diff
@@ -45,18 +45,24 @@ export class ComplianceAndEnforcementService {
     return this.mapper.mapArray(entity, ComplianceAndEnforcement, ComplianceAndEnforcementDto);
   }
 
-  async fetchByUuid(
-    uuid: string,
+  async fetchById(
```

Comment:
This is unrelated to ticket, just cleaning this up. There was a lot of duplicated code here, and they had mismatches. `fetchByUuid` was never used.